orttraining/orttraining/training_ops/cpu/math/scale.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "scale.h"
+#include "core/providers/common.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+#define REGISTER_SCALE_KERNEL_TYPED(T, ScaleT)                              \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                            \
+      Scale,                                                                \
+      kMSDomain,                                                            \
+      1,                                                                    \
+      T##_##ScaleT,                                                         \
+      kCpuExecutionProvider,                                                \
+      KernelDefBuilder()                                                    \
+          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>())            \
+          .TypeConstraint("ScaleT", DataTypeImpl::GetTensorType<ScaleT>()), \
+      Scale<T, ScaleT>);
+
+template <typename T, typename ScaleT>
+Status Scale<T, ScaleT>::Compute(OpKernelContext* context) const {
+  const Tensor* scale_tensor = context->Input<Tensor>(1);
+  ORT_ENFORCE(scale_tensor->Shape().Size() == 1, "Scale input should have a single value.");
+  const float scale_value = static_cast<float>(*scale_tensor->Data<ScaleT>());
+  ORT_ENFORCE(scale_value != 0.0f, "Scale value must not be 0.");
+  const T inverse_scale_value = static_cast<T>(1.0f / scale_value);
```

Comment:
It's surprising to have the default behavior as Div by Scale. 
Maybe introduce an attribute to specify the behavior, which it's Mul or Div by scale